CHANGELOG.md

```diff
@@ -4,7 +4,21 @@ This file summarizes notable changes introduced in aktualizr version. It roughly
 
 Our versioning scheme is `YEAR.N` where `N` is incremented whenever a new release is issued. Thus `N` does not necessarily map to months of the year.
 
-## [??? (unreleased)]
+## [upcoming release]
+
+### Special considerations
+
+As a result of changes to the IP/POSIX Secondary protocol (see below), users of these Secondaries will need to take special care when upgrading their devices. The new version of aktualizr (e.g. the Primary) is backwards compatible and will work with both old and new versions of the protocol. However, the aktualizr-secondary is *not*. This means that if you are upgrading a device with IP/POSIX Secondaries, you should update the Primary ECU first, and if that is successful, then update your Secondaries.
```

Comment:
```suggestion
As a result of changes to the IP/POSIX Secondary protocol (see below), users of these Secondaries will need to take special care when upgrading their devices. The new version of aktualizr (for example, the Primary) is backwards compatible and will work with both old and new versions of the protocol. However, the aktualizr-secondary is *not*. This means that if you are upgrading a device with IP/POSIX Secondaries, you should update the Primary ECU first, and if that is successful, update your Secondaries.
```

Reply:
`e.g.` is correct, but as a general consideration, we should omit Latin words in the documentation.

Reply:
Actually it turns out I meant "i.e." I often mix those up. In this case, I meant that aktualizr = the Primary. I think I see how I can rephrase to make it clearer.

Reply:
Also: why remove the "then"? I don't think it is strictly necessary, but I think the if-then construction is clearer than just "if" alone.

Reply:
i.e. => that is

As to "then", yep, it's not strictly necessary. It's also clear without "then". It was just a small suggestion to make the sentence shorter. But if you decide to go for "if-then", I'll have nothing against that :)